Sources/Public/Animation/AnimationView.swift

```diff
@@ -20,6 +20,8 @@ public enum LottieBackgroundBehavior {
   case pauseAndRestore
   /// Stops the animation and sets it to the end of its current play time. The completion block is called.
   case forceFinish
+  /// Does nothing when the animation moves to the background
+  case doNothing
```

Comment:
Can you clarify that this is the default for the core animation engine? And then can you clarify that in the `pause` case with the core animation rendering engine that it would introduce unnecessary overhead? I think it would be helpful to have more discussion in these to help users make sensible choices from a performance perspective